tests/kernel/queue/src/main.c

```diff
@@ -8,24 +8,66 @@
 #include "test_queue.h"
 
 #ifndef CONFIG_USERSPACE
-#define dummy_test(_name)		\
-	static void _name(void)		\
-	{				\
-		ztest_test_skip();	\
-	}
-
-dummy_test(test_queue_supv_to_user);
-dummy_test(test_queue_alloc_prepend_user);
-dummy_test(test_queue_alloc_append_user);
-dummy_test(test_auto_free);
-dummy_test(test_queue_init_null);
-dummy_test(test_queue_alloc_append_null);
-dummy_test(test_queue_alloc_prepend_null);
-dummy_test(test_queue_get_null);
-dummy_test(test_queue_is_empty_null);
-dummy_test(test_queue_peek_head_null);
-dummy_test(test_queue_peek_tail_null);
-dummy_test(test_queue_cancel_wait_error);
+
+ZTEST(queue_api, test_queue_supv_to_user)
+{
+	ztest_test_skip();
```

Comment:
Seems like it would be cleaner to just add a `if (IS_ENABLED(CONFIG_USERSPACE)) { ztest_test_skip(); }` to the affected tests rather than expand out five lines of code for every one, but that can be done later as cleanup if we care.

Reply:
Hi @andyross,  thanks for correcting, I have updated the PR.